helm_deploy/values-preprod.yaml

```diff
@@ -19,7 +19,7 @@ ingress:
 env:
   PRISON_URI_ROOT: "https://api-preprod.prison.service.justice.gov.uk"
   AUTH_URI_ROOT: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
-  JAVA_OPTS: "-Xmx1024m"
+  JAVA_OPTS: "-Xmx512m"
```

Comment:
Did a heap dump on the servers and discovered that the heap wasn't that close to 512 - so put it back to 512 as it also looked like with all the other memory settings it might potentially get over 2Gb and be killed by kubernetes.